src/Slices/ServiceInventory/UI/Presenters/InstanceExpertActionPresenter.ts

```diff
@@ -0,0 +1,45 @@
+import React, { ReactElement } from "react";
+import { ServiceModel } from "@/Core";
+import { ActionPresenter, ServiceInstanceForAction } from "@/UI/Presenters";
+import { ExpertActions } from "../Components/ExpertActions";
+
+export class InstanceExpertActionPresenter implements ActionPresenter {
+  constructor(
+    private readonly instances: ServiceInstanceForAction[],
+    private readonly serviceEntity: ServiceModel
+  ) {}
+
+  private getInstanceForId(id: string): ServiceInstanceForAction | undefined {
+    return this.instances.find((instance) => instance.id === id);
+  }
+
+  getForId(id: string): ReactElement | null {
```

Comment:
get what for id? I kind of remember making a comment like that already, I think i asked for a comment too, that shortly explains the presenter

Reply:
it was changed, I don't know why it isn't showing here, but when I'm in that file there is a comment for that function